recipes/libev/all/conanfile.py

```diff
@@ -33,6 +34,10 @@ class LibevConan(ConanFile):
     def _settings_build(self):
         return getattr(self, "settings_build", self.settings)
 
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
+        copy(self, "config.h", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
```

Comment:
I believe it would be neater to export these files in base folder to not mix up these files with upstream source files, and adapt CMakeLists and configure() method accordingly.

Reply:
I'm not sure that I understand what you mean and how to do that. Can you provide some examples, maybe?

Reply:
I see the same usage here https://github.com/conan-io/conan-center-index/blob/master/recipes/qtxlsxwriter/all/conanfile.py#32

Reply:
@Jihadist The idea is copying CMakeLists.txt to export source folder, but not in "src", like `copy(self, "CMakeLists.txt", self.recipe_folder, self.export_sources_folder)`
So you include the source folder in your cmake file. It avoid mixing source with your created file. As real example we have:
- https://github.com/conan-io/conan-center-index/tree/master/recipes/geotrans/all
- https://github.com/conan-io/conan-center-index/tree/master/recipes/vcglib/all
- https://github.com/conan-io/conan-center-index/tree/master/recipes/tgc/all
- https://github.com/conan-io/conan-center-index/blob/master/recipes/bzip2/all

Could you please update to not copying your cmake file with the src folder.

Reply:
Ok, i changed it